.gitignore

```diff
@@ -4,3 +4,4 @@ node_modules
 .idea
 app-dist
 dist
+.nuxt
```

Comment:
Prettier :p

Reply:
?

Reply:
If checking git diff it is removing last empty line:

![image](https://user-images.githubusercontent.com/5158436/90293538-9d0ac700-de84-11ea-8ad1-94878dd74b3c.png)

nothing important just  to say since we have it by default for most of repos [example](https://github.com/nuxt/nuxt.js/blob/dev/.editorconfig#L10)

Reply:
ohh I am sure that was just me who didn't add a last line when I added .nuxt as don't have prettier installed on this project and I am so used to just saving and it automatically doing it all for me and on this project it's all manual. its killing me.

Reply:
I don't know what is it but i'm sure you are not only one (since recently see with other PRs too) but would be nice if finding out what is causing :D

Reply:
maybe add the editor config to the pr and that way it will enforce it on everyone who uses the project?

Reply:
adding file would be nice idea!